content/manager_webui/templates-mgmt.md

```diff
@@ -59,7 +59,7 @@ In case a user fits more than one template in a specific tenant (if the user has
 ### Creating a new page
 
 After clicking the “Create page”  button and specifying the name of the new page, you will see the new page appear in the page’s list, and a Page Id will be automatically created for it. You will then be able to choose or edit it from the actions menu to its right. 
-Upon choosing the edit option, the page will be open in “edit mode”, in which you can add widgets and edit their properties, order and appearance.
+Upon choosing the edit option, the page will be open in “edit mode”, in which you can add widgets and edit their properties, order and appearance.You can also change the page name by clicking on the name appearing on the top (after "Template management/". 
```

Comment:
- Add space between . and You
- appearing > shown
- Do we need the / ?
- Close parentheses